tests-e2e/core/argocd_gitopsengine_instance_test.go

```diff
@@ -0,0 +1,140 @@
+package core
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/redhat-appstudio/managed-gitops/backend-shared/db"
+	"github.com/redhat-appstudio/managed-gitops/backend/util"
+	"github.com/redhat-appstudio/managed-gitops/tests-e2e/fixture"
+	"github.com/redhat-appstudio/managed-gitops/tests-e2e/fixture/k8s"
+	apps "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var _ = Describe("ArgoCD instance via GitOpsEngineInstance Operations Test", func() {
+
+	const (
+		argocdNamespace = fixture.NewArgoCDInstanceNamespace
+	)
+
+	Context("ArgoCD instance gets created from an operation's gitopsEngineInstance resource-type", func() {
+
+		BeforeEach(func() {
+			By("Delete old namespaces, and kube-system resources")
+			Expect(fixture.EnsureCleanSlate()).To(Succeed())
+
+		})
+
+		It("ensures that a standalone ArgoCD gets created successfully when an operation CR of resource-type GitOpsEngineInstance is created", func() {
+			err := unsafeCleanDBWhereIDisAutoGenerated()
```

Comment:
Do you by chance know which other E2E test was interfering with this test?

Reply:
I observed two things, either a test was affecting the current test or the current test was affecting the other tests. In sequence, whichever was being tested _before_ and _after_ the current test should be the answer here. I haven't seen a recurring pattern of tests on ci but I might be wrong.